### PR TITLE
feat: add Supabase Cron docs

### PR DESCRIPTION
**Pull request by w3b6x9**

### CODE REVIEW

File: apps/docs/content/guides/cron.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: 'Cron'
+subtitle: 'Schedule Recurring Jobs with Cron Syntax in Postgres'
+---
+
+Supabase Cron is a Postgres Module that simplifies scheduling recurring Jobs with cron syntax and monitoring Job runs inside Postgres.
+
+Cron Jobs can be created via SQL or Cron interface inside of Supabase Dashboard and can run anywhere from every second to once a year depending on your use case.
```
Comment (charislam) [inline apps/docs/content/guides/cron.mdx:8]:
```suggestion
Cron Jobs can be created via SQL or the Cron interface inside of Supabase Dashboard and can run anywhere from every second to once a year depending on your use case.
```

File: apps/docs/content/guides/cron.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Every Job can run SQL snippets or database functions with zero network latency or make an HTTP request, such as invoking a Supabase Edge Function, with ease.
+
+<Admonition type="note">
+
+For the best performance, we recommend no more than 8 Jobs run concurrently and each Job should run no more than 10 minutes.
```
Comment (charislam) [inline apps/docs/content/guides/cron.mdx:14]:
```suggestion
For best performance, we recommend no more than 8 Jobs run concurrently. Each Job should run no more than 10 minutes.
```

File: apps/docs/content/guides/cron.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Under the hood, Supabase Cron uses the [`pg_cron`](https://github.com/citusdata/pg_cron) Postgres database extension which is the scheduling and execution engine for your Jobs.
+
+<Admonition type="note">
+
+`pg_cron` is not fully supported on Fly Postgres. Learn more about [Fly Postgres limitations](/docs/guides/platform/fly-postgres#limitations).
+
+</Admonition>
+
+The extension creates a `cron` schema in your database and all Jobs are stored on the `cron.job` table and every Job's run and its status is recorded on the `cron.job_run_details` table.
```
Comment (charislam) [inline apps/docs/content/guides/cron.mdx:28]:
```suggestion
The extension creates a `cron` schema in your database and all Jobs are stored on the `cron.job` table. Every Job's run and its status is recorded on the `cron.job_run_details` table.
```

File: apps/docs/content/guides/cron.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+This is paired with a friendly interface in the Supabase Dashboard to schedule your Jobs as well as real-time monitoring of current and historical runs.
```
Comment (charislam) [inline apps/docs/content/guides/cron.mdx:30]:
```suggestion
The Supabase Dashboard provides and interface for you to schedule Jobs and monitor Job runs. You can also do the same with SQL.
```

File: apps/docs/content/guides/cron/create-cron-job.mdx
Comment (charislam) [inline apps/docs/content/guides/cron/create-cron-job.mdx:1]: @w3b6x9 do you still want the `create-cron-job` and `edit-delete-cron-job` files? I think these are the ones I wrote and they're not in your nav, so I wasn't sure if you wanted to delete them

File: apps/docs/content/guides/cron/install.mdx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="dashboard"
+  queryGroup="database-method"
+>
+<TabPanel id="dashboard" label="Dashboard">
+
+1. Go to the [Cron Postgres Module](https://supabase.com/dashboard/project/_/integrations/cron/overview) under Integrations in the Dashboard.
```
Comment (charislam) [inline apps/docs/content/guides/cron/install.mdx:16]:
```suggestion
1. Go to the [Cron Postgres Module](/dashboard/project/_/integrations/cron/overview) under Integrations in the Dashboard.
```

File: apps/docs/content/guides/cron/install.mdx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+2. Enable `pg_cron` extension.
```
Comment (charislam) [inline apps/docs/content/guides/cron/install.mdx:17]:
```suggestion
2. Enable the `pg_cron` extension.
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="dashboard-schedule-job"
+  queryGroup="database-method"
+>
+<TabPanel id="dashboard-schedule-job" label="Dashboard">
+
+1. Go to the [Jobs](https://supabase.com/dashboard/project/_/integrations/cron/jobs) section to schedule your first Job.
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:24]:
```suggestion
1. Go to the [Jobs](/dashboard/project/_/integrations/cron/jobs) section to schedule your first Job.
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+2. Click on `Create job` button or navigate to the new Cron Job form [here](https://supabase.com/dashboard/project/_/integrations/cron/jobs?dialog-shown=true).
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:25]:
```suggestion
2. Click on `Create job` button or navigate to the new Cron Job form [here](/dashboard/project/_/integrations/cron/jobs?dialog-shown=true).
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="dashboard-unschedule-job"
+  queryGroup="database-method"
+>
+<TabPanel id="dashboard-unschedule-job" label="Dashboard">
+
+1. Go to the [Jobs](https://supabase.com/dashboard/project/_/integrations/cron/jobs) section and find the Job you'd like to unschedule.
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:134]:
```suggestion
1. Go to the [Jobs](/dashboard/project/_/integrations/cron/jobs) section and find the Job you'd like to unschedule.
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 10]
```diff
@@ -0,0 +1,294 @@
...
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="dashboard-delete-job"
+  queryGroup="database-method"
+>
+<TabPanel id="dashboard-delete-job" label="Dashboard">
+
+1. Go to the [Jobs](https://supabase.com/dashboard/project/_/integrations/cron/jobs) section and find the Job you'd like to delete.
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:168]:
```suggestion
1. Go to the [Jobs](/dashboard/project/_/integrations/cron/jobs) section and find the Job you'd like to delete.
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 11]
```diff
@@ -0,0 +1,294 @@
...
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="dashboard-runs-job"
+  queryGroup="database-method"
+>
+<TabPanel id="dashboard-runs-job" label="Dashboard">
+
+1. Go to the [Jobs](https://supabase.com/dashboard/project/_/integrations/cron/jobs) section and find the Job you want to see the runs of.
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:199]:
```suggestion
1. Go to the [Jobs](/dashboard/project/_/integrations/cron/jobs) section and find the Job you want to see the runs of.
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 12]
```diff
@@ -0,0 +1,294 @@
...
+2. Click on `History` button next to the Job name.
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:200]:
```suggestion
2. Click on the `History` button next to the Job name.
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 13]
```diff
@@ -0,0 +1,294 @@
...
+  *
+from cron.job_run_details
+where jobid = (select jobid from cron.job where jobname = 'permanent-cron-job-name')
+order by start_time desc
+limit 10;
+```
+
+<Admonition type="warning">
+
+The records in `cron.job_run_details` table are not cleaned up automatically nor when Jobs are unscheduled which will take up disk space in your database.
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:216]:
```suggestion
The records in the `cron.job_run_details` table are not cleaned up automatically. They are also not removed when jobs are unscheduled, which will take up disk space in your database.
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 14]
```diff
@@ -0,0 +1,294 @@
...
+
+Vacuum every day at 3:00am (GMT):
+
+```sql
+select cron.schedule('nightly-vacuum', '0 3 * * *', 'VACUUM');
+```
+
+### Call a database function every 5 minutes
+
+Create a [`hello_world()`](https://supabase.com/docs/guides/database/functions?language=sql#simple-functions) database function and then call it every 5 minutes:
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:247]:
```suggestion
Create a [`hello_world()`](/docs/guides/database/functions?language=sql#simple-functions) database function and then call it every 5 minutes:
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 15]
```diff
@@ -0,0 +1,294 @@
...
+          body:=jsonb_build_object('time', now() ),
+          timeout_milliseconds:=5000
+      ) as request_id;
+    $$
+  );
+```
+
+<Admonition type="note">
+
+This requires [`pg_net` extension](/docs/guides/database/extensions/pg_net) to be enabled.
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:284]:
```suggestion
This requires the [`pg_net` extension](/docs/guides/database/extensions/pg_net) to be enabled.
```

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 16]
```diff
@@ -0,0 +1,294 @@
...
+
+</Admonition>
+
+## Caution: Scheduling System Maintenance
+
+Be extremely careful when setting up Jobs for system maintenance tasks as they can have unintended consequences.
+
+For instance, scheduling a command to terminate idle connections with `pg_terminate_backend(pid)` can disrupt critical background processes like nightly backups. Often, there is an existing Postgres setting e.g. `idle_session_timeout` that can perform these common maintenance tasks without the risk.
```
Comment (charislam) [inline apps/docs/content/guides/cron/quickstart.mdx:292]:
```suggestion
For instance, scheduling a command to terminate idle connections with `pg_terminate_backend(pid)` can disrupt critical background processes like nightly backups. Often, there is an existing Postgres setting, such as `idle_session_timeout`, that can perform these common maintenance tasks without the risk.
```

File: apps/docs/content/guides/database/extensions/pg_cron.mdx [hunk 17]
```diff
@@ -4,191 +4,4 @@ title: 'pg_cron: Job Scheduling'
...
-<Admonition type="note">
-
-The records in cron.job_run_details are not cleaned automatically which will take up disk space in your database.
-
-</Admonition>
-
-## Resources
-
-- [`pg_cron` GitHub Repository](https://github.com/citusdata/pg_cron)
+These docs have been moved to [Supabase Cron](https://supabase.com/docs/guides/cron).
```
Comment (charislam) [inline apps/docs/content/guides/database/extensions/pg_cron.mdx:7]:
```suggestion
See the [Supabase Cron docs](/docs/guides/cron).
```

File: apps/docs/content/guides/cron/create-cron-job.mdx
Comment (w3b6x9) [reply apps/docs/content/guides/cron/create-cron-job.mdx:1]: @charislam yea i was trying to delete them. if there still on this branch i'll seek to remove now.

Comment (w3b6x9) [reply apps/docs/content/guides/cron/create-cron-job.mdx:1]: done: https://github.com/supabase/supabase/pull/30822/commits/626e970822dc2e43b374f41c157fb9642ae3382c

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 18]
```diff
@@ -0,0 +1,322 @@
...
+  size="small"
+  type="underlined"
+  defaultActiveId="dashboard-runs-job"
+  queryGroup="database-method"
+>
+<TabPanel id="dashboard-runs-job" label="Dashboard">
+
+1. Go to the [Jobs](/dashboard/project/_/integrations/cron/jobs) section and find the Job you want to see the runs of.
+2. Click on the `History` button next to the Job name.
+
```
Comment (w3b6x9) [inline apps/docs/content/guides/cron/quickstart.mdx:229]: @olirice drop image here pls

**Review by charislam — APPROVED** [review]

**Pull request merged**